### PR TITLE
Add references to 'Writing documentation' page

### PR DESCRIPTION
**Pull request by michalmuskala**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -6,6 +6,8 @@
 
 ExDoc is a tool to generate documentation for your Elixir projects. In case you are looking for documentation for Elixir itself, [check out Elixir's website][elixir-lang].
 
+For documentation on writing documentation [check out Elixir's documentation][hex-writing-docs].
```
Comment (eksperimental) [inline README.md]: three times the word "documentation" in a sentence is a bit too much I think.
How about:
> To learn more about how to document this project [check out Elixir's documentation][hex-writing-docs]

Comment (eksperimental) [inline README.md]: sorry for some reason I thought it this PR was submitted to Elixir.

how about:
> To learn about how to document your projects check out [Elixir's writing documentation page][hex-writing-docs]

Comment (michalmuskala) [inline README.md]: I have to admit, it was kind of intentional 😈 , but I can see how it can be confusing

**Comment by josevalim** [comment]
:heart: :green_heart: :blue_heart: :yellow_heart: :purple_heart:

**Pull request merged**